src/conversion.jl

```diff
@@ -195,6 +195,20 @@ function Expr(x::EXPR)
         Expr(:string, Expr.(x.args[2:end])...)
     elseif x.args === nothing
         Expr(Symbol(lowercase(String(x.head))))
+    elseif x.head === :parameters
+        if length(x.args) > 1 && any(a -> a.head === :parameters, x.args)
+            ordered_args = EXPR[]
+            for arg in x.args
+                if arg.head === :parameters
+                    pushfirst!(ordered_args, arg)
+                else
+                    push!(ordered_args, arg)
+                end
+            end
+            Expr(:parameters, Expr.(ordered_args)...)
+        else
+            Expr(:parameters, Expr.(x.args)...)
+        end
```

Comment:
Not a priority right now but we shouldn't need to special case conversion so should look to fix the underlying inconsistency w/ the scheme parser

Reply:
Yeah, that's basically what I reverted here. Originally I had the right order in the `EXPR`, but that failed during `iterate`, so this was easier for me than wrapping my head around how iteration works :P